client/src/app/components/stock-detail-header/stock-detail-header.component.scss

```diff
@@ -21,7 +21,6 @@
         border: 1px solid black;
         height: 50px;
         width: 80px;
-        background: url('https://d1yjjnpx0p53s8.cloudfront.net/styles/logo-thumbnail/s3/0021/6847/brand.gif?itok=u0iVoArk');
```

Comment:
This is the logo of the company?

Reply:
This was the logo for air canada. In the story before we hardcoded all of the data just to see something show up, but in this story we use real data, which is why I removed this

Reply:
Ah ok perfect, thank you!